FrontEndApp/v1-react/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Please change the name and short name here to Auto-DL Generator Frontend and Auto-DL or something on these lines